mw-webapp/src/styles/_variables.scss

```diff
@@ -12,4 +12,11 @@ $link-active: rgb(255, 149, 0);
 $header-background: rgb(85, 122, 149);
 
 // Title
-$title-padding: 10px;
\ No newline at end of file
+$title-padding: 10px;
+
+// font size
```

Comment:
Start comment from uppercase

Reply:
Fixed.

Reply:
Should linter fix it automatically? Please check configuration